src/main/java/org/llorllale/cactoos/matchers/ScalarHasValue.java

```diff
@@ -38,6 +38,10 @@
  *
  * @param <T> Type of result
  * @since 0.2
+ * @todo #75:30min All the matchers should extend MatcherEnvelope and
```

Comment:
@borysfan I didn't understand this PDD. `ScalarHasValue` not extends `MatcherEnvelope`. Why is it *an example of a satifsfatory result*?

Reply:
@fabriciofx I used @Iprogrammerr branch and I didn't notice this PDD. PDD has been removed.